File: Bubble sort/bubble_sort.py
def bubble_sort(array) :

    for x in range(len(array)-1) :

        for y in range(len(array)-1-x) :

            if array[y] > array[y+1] :
                temp = array[y]
                array[y] = array[y+1]
                array[y+1] = temp
                
    return array

array = [5,1,7,2,9,0]
sorted_array = bubble_sort(array)
print("Array after sorting is ",sorted_array)

File: Bubble sort/bubble_sort.cpp
#include <stdio.h>
#include<stdlib.h>

void Bubble(int A[],int n)
{
int i,j,temp;

for(i=0;i<n-1;i++)
{

for(j=0;j<n-i-1;j++)
{

if(A[j]>A[j+1])
{
temp = A[j];
A[j] = A[j+1];
A[j+1] = temp;
}

}
}
}

int main()
{
int A[]={5,1,7,2,9,0},i;
Bubble(A,sizeof(A)/sizeof(int));
printf("Array after sorting is [ ");
for(i=0;i<sizeof(A)/sizeof(int);i++)
printf("%d ",A[i]);
printf("]\n");
return 0;
}